sycl/test-e2e/Basic/buffer/subbuffer.cpp

```diff
@@ -1,6 +1,7 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
+// See https://github.com/intel/llvm/issues/15151
```

Comment:
Not exactly reenabling. I'm okay with it, just noting it in case it was involuntarily added to this PR.

Reply:
I originally tried to reenable it, but it seemed to still fail on Gen12. I will make this clear in the description of the PR.